python/ray/util/client/server/dataservicer.py

```diff
@@ -19,6 +20,20 @@
 logger = logging.getLogger(__name__)
 
 
+def fill_queue(grpc_input_generator, output_queue) -> None:
+    """
+    Pushes incoming requests to a shared output_queue.
+    """
+    try:
+        for req in grpc_input_generator:
+            output_queue.put(req)
+    except grpc.RpcError as e:
+        logger.debug("closing dataservicer reader thread "
```

Comment:
```suggestion
        logger.exception("closing dataservicer reader thread "
```

Reply:
@AmeerHajAli Most places that catch `grpc.RpcError`s just Debug log. I would prefer to keep this the same as all those calls.